calculator.py

```diff
@@ -0,0 +1,24 @@
+def sum(a: int, b: int) -> int:
+    return a + b
+
+def diff(a: int, b: int)-> int:
+    return a-b
+
+def mul(a: int, b: int)-> int:
+    return a*b
+
+def div(a: int, b:int)-> int:
```

Comment:
Mi sembra tutto ok, una piccola nota è che potresti aggiungere in questa funzione un controllo per assicurarti che b non sia zero e in quel caso ritornare magari None :)